01-Login/README.md

```diff
@@ -68,4 +75,1059 @@ If you have found a bug or if you have a feature request, please report them at
 
 ## License
 
-This project is licensed under the MIT license. See the [LICENSE](LICENSE.txt) file for more info.
+This project is licensed under the MIT license. See the [LICENSE](../LICENSE) file for more info.
+
+# Tutorial
+This tutorial will help you get up and running the React app using the new SDK.
+
+## Setting Up the Application
+
+### Installing initial dependencies
+After create a new React app using `create-react-app` we still need to install some dependencies that doesn't come with the boilerplate. These dependencies will help us with things like routing, styling and font icons. But, before that we need to move the current `dependencies` to `devDependencies` this is because we will be using a server to run the built app.
+
+```json
+- "dependencies": {
++ "devDependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "2.1.8"
+  }
+```
+
+After that we can install the new dev dependencies.
+
+```bash
+npm i -D react-router-dom highlight.js reactstrap samples-bootstrap-theme \
+@fortawesome/fontawesome-svg-core @fortawesome/free-solid-svg-icons @fortawesome/react-fontawesome
+```
+### Create the Home view and initial components
+We are going to create a serie of components in order to build our initial welcome page. Create a folder called `views` and then a Compoment called `Home.js`. This component will show a welcome message and some information. 
+
+```jsx
+// src/views/Home.js
+import React, { Fragment } from "react";
+
+import Hero from "../components/Hero";
+import Content from "../components/Content";
+
+const Home = () => (
+  <Fragment>
+    <Hero />
+    <hr />
+    <Content />
+  </Fragment>
+);
+
+export default Home;
+```
+
+As you can see our `Home.js` component is build with other component pieces. Create a folder called `Compoments` and inside create two new components called `Hero.js`, and `Content.js`.
+
+For our `Hero.js` compoment we need to move the `logo.svg` inside a folder called `assets`.
+
+```jsx
+// src/components/Hero.js
+import React from "react";
+
+import logo from "../assets/logo.svg";
+
+const Hero = () => (
+  <div className="text-center hero">
+    <img className="mb-3 app-logo" src={logo} alt="React logo" />
+    <h1 className="mb-4">React.js Sample Project</h1>
+
+    <p className="lead">
+      This is a sample application that demonstrates an authentication flow for
+      an SPA, using <a href="https://reactjs.org">React.js</a>
+    </p>
+  </div>
+);
+
+export default Hero;
+```
+
+Since we want to load the content more dinamically, we are going to create a file called `contentData.js` inside a new folder called `utils`. This data will be used inside the `Content.js` component.
```

Comment:
This is a detail that is specific to the sample application and not something the reader of the quickstart should really deal with. For the tutorial, it's fine to focus on the authentication parts and not worry about the visual styling and content at all.

So basically this paragraph, the following code snippet and the content component could be dropped from the tutorial. If the reader wants to check out how that is done, they can refer to the sample.